test/integration/verify/controls/google_billing_project_billing_info.rb

```diff
@@ -15,6 +15,8 @@
 title 'Test GCP google_billing_project_billing_info resource.'
 
 gcp_project_id = attribute(:gcp_project_id, default: 'gcp_project_id', description: 'The GCP project identifier.')
+gcp_billing_account = attribute(:gcp_billing_account, default: 'gcp_billing_account', description: 'The GCP billing account name.')
+gcp_organization_id = attribute(:gcp_organization_id, default: gcp_organization_id, description: 'The identifier of the organization')
```

Comment:
Hey @slevenick the build failure is due to `gcp_organization_id` being missing but seeing this test now I'd suggest to remove this otherwise unused variable and update the `only_if` to:

```
  only_if { gcp_enable_privileged_resources.to_i == 1 && gcp_billing_account != ''}
```

Reply:
Yeah, that makes sense. Currently Magic Modules only knows how to check the `gcp_enable_privileged_resources` and `gcp_organization_id` in that skip check, so it will take a couple edits to get that in there

Reply:
I'm happy to proceed with this for now as it fixes the build and rework the `only_if` at a later stage.